Riot/Modules/Room/TimelineCells/Styles/Plain/Cells/TextMessage/Incoming/Clear/RoomIncomingTextMsgBubbleCell.m

```diff
@@ -27,12 +27,7 @@ - (void)customizeTableViewCellRendering
 {
     [super customizeTableViewCellRendering];
     
-<<<<<<< HEAD
-    self.userNameLabel.textColor = ThemeService.shared.theme.userNameColors[0];
-    self.messageTextView.tintColor = ThemeService.shared.theme.tintColor;
-=======
     [self updateUserNameColor];
```

Comment:
Should we apply colors in this method ?

Reply:
In fact, `MXKRoomBubbleTableViewCell+Riot`.`updateUserNameColor` method body is fully commented.

Reply:
https://github.com/tchapgouv/tchap-ios/pull/744#discussion_r1105604219

Reply:
Done in following commit.